### DISCUSSION TITLE
timestamp for diagnoses_icd

### DISCUSSION THREAD

**Discussion by sunukp** [MIT-LCP/mimic-code#1202] [MIMIC-III]
The diagnoses_ICD table doesn't contain any timestamps (or at least for MIMIC-III it didn't). I am interested in finding out when a specific condition was diagnosed. For example, if a patient was diagnosed as having meningitis in the diagnoses_ICD table, is there a way to tell if s/he had it before the ICU or has s/he developed it during her/his ICU stay?

**Comment by alistairewj** [comment]
Diagnoses do not have a timestamp. They are coded after hospital discharge based on a retrospective review of the patient's clinical notes by a professional coder. There isn't a consistent approach to identifying when a condition occurred - but depending on the condition, many will assume a time period (e.g. for some chronic conditions, it's reasonable to assume it was present on admission to the hospital).
